e2e/cypress/integration/integrations/outgoing_webhook/delete_outgoing_webhook_spec.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Integrations', () => {
+    let testTeam;
+    let testChannel;
+    let testUser;
+    let outgoingWebhook;
+
+    //const callbackUrl = `${Cypress.env().webhookBaseUrl}/post_outgoing_webhook`;
+
+    before(() => {
+        const callbackUrl = `${Cypress.env().webhookBaseUrl}/post_outgoing_webhook`;
+
+        cy.requireWebhookServer();
+
+        // # Create test team, channel, and webhook
+        cy.apiInitSetup().then(({team, channel, user}) => {
+            testTeam = team.name;
+            testChannel = channel.name;
+            testUser = user;
+
+            const newOutgoingHook = {
+                team_id: team.id,
+                display_name: 'New Outgoing Webhook',
+                trigger_words: ['testing'],
+                callback_urls: [callbackUrl],
+            };
+
+            cy.apiCreateWebhook(newOutgoingHook, false).then((hook) => {
+                outgoingWebhook = hook;
+
+                cy.apiGetWebhook(outgoingWebhook.id, false).then((res) => {
+                    expect(res.status).equal(200);
+                    expect(res.body.id).equal(outgoingWebhook.id);
+                });
+            });
+
+            cy.apiLogin(user);
+        });
+    });
+
+    it('MM-T617 Delete outgoing webhook', () => {
+        // # Confirm outgoing webhook is working
+        cy.visit(`/${testTeam}/channels/${testChannel}`);
+        cy.uiPostMessageQuickly('testing', {timeout: TIMEOUTS.ONE_MIN});
```

Comment:
There's no second argument for [cy. uiPostMessageQuickly](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/ui_commands.js#L94). Also, this can simply,
```suggestion
        cy.postMessage('testing');
```